Docs/MoltenVK_Runtime_UserGuide.md

```diff
@@ -328,6 +329,7 @@ In addition to core *Vulkan* functionality, **MoltenVK**  also supports the foll
 - `VK_AMD_shader_trinary_minmax` *(requires Metal 2.1)*
 - `VK_IMG_format_pvrtc` *(requires Apple GPU)*
 - `VK_INTEL_shader_integer_functions2`
+- `VK_NV_fragment_shader_barycentric`
```

Comment:
For consistency with `KHR` entry, please add  *(requires Metal 2.2 on Mac or Metal 2.3 on iOS)* here too.